pkg/frontend/querymiddleware/roundtrip_test.go

```diff
@@ -320,16 +415,9 @@ func TestTripperware_Metrics(t *testing.T) {
 			require.NoError(t, err)
 			require.Equal(t, 200, resp.StatusCode)
 
-			body, err := io.ReadAll(resp.Body)
-			require.NoError(t, err)
-			require.Equal(t, `{"status":""}`, string(body))
```

Comment:
Note to reviewers: removed because the response content is different based on the request type. I don't think it's particularly important to assert on it, given this test is specifically designed to assert tracked metrics.